stable/admin/README.md

```diff
@@ -186,6 +186,8 @@ The following tables list the configurable parameters for the `admin` option of
 | `podAnnotations` | Annotations to pass through to the Admin pod | `nil` |
 | `tde.secrets` | Transparent Data Encryption secret names used for different databases | `{}` |
 | `tde.storagePasswordsDir` | Transparent Data Encryption storage passwords mount path | `/etc/nuodb/tde` |
+| `evicted.servers` | A list of evicted servers excluded from RAFT consensus. Used during disaster recovery. | `[]` |
```

Comment:
Why not `admin.evictedServers`? Do we plan to add more keys under `admin.evicted` variable?

Reply:
+1